plugins/pipeline_plugin/operators/RCloneOperator.py

```diff
@@ -0,0 +1,48 @@
+import os
+import tempfile
+from airflow.utils.decorators import apply_defaults
+from pipeline_plugin.config import config
+from pipeline_plugin.utils.google_cloud_storage import download_file
+from pathlib import Path
+
+from pipeline_plugin.operators.BaseMapActionOperator import MapActionOperator
+import subprocess
+
+
+def sync_from_gcp_to_gdrive():
+    gcp_path = config.get_data_bucket_name()
+    gdrive_folder_id = config.get_google_drive_output_folder_id()
+
+    _, service_account_auth_path = tempfile.mkstemp(suffix='.json')
+    gcsc = GoogleCloudStorageClient()
+
+    try:
+        print(tmp_file_path)
+
+        gcsc.download_file_from_gcs(
+            bucket_name: config.get_rclone_service_account_auth_bucket(),
+            source_blob: config.get_rclone_service_account_auth_file(),
+            destination_filename=service_account_auth
```

Comment:
I think this should be `service_account_auth_path`

Reply:
You think right.